lib/util/git/index.ts

```diff
@@ -155,7 +201,7 @@ export async function validateGitVersion(): Promise<boolean> {
   let version: string;
   const globalGit = simpleGit.default();
   try {
-    const raw = await globalGit.raw(['--version']);
+    const raw = await gitRetry(() => globalGit.raw(['--version']));
```

Comment:
I think this shouldn't be retried, if this fails there is something really wrong.